### ISSUE TITLE
Full dps for Gems within the same linked sockets.

### ISSUE THREAD

**Issue by EdwoG** [LocalIdentity/PathOfBuilding#3564]
### Check for duplicates

- [x] I've checked for duplicate issues by using the search function of the [issue tracker](https://github.com/PathOfBuildingCommunity/PathOfBuilding/issues)

### Is your feature request related to a problem?

Frostblade has the 2 components, Melee and Projectile.

With hydrosphere I hit bosses with both components.

Its hard to find out optimal gems, since im only able to Sort for the dps of only 1 of the components.

And including both in Full dps still sorts the gems dps, while its only applied to 1 of the gems, not both.

### Describe the solution you'd like

Have support gems apply to both components and sort by full dps

**Comment by Nostrademous** [comment]
So in other words, you would like to be able to combine numerous `skill parts` of a single gem when considering support gem recommendations, correct?

**Comment by EdwoG** [comment]
yes, i guess it would solve it, if i was able to check both for fulldps
i believe right now I can only check frostblades
and then i choose a skill part on the left side, which to consider, but i needed both for the build

**Comment by Nostrademous** [comment]
Okay, so the better way to solve it probably is to create a new (3rd) skill part which combines both.
